turbinia/evidence.py

```diff
@@ -566,17 +647,17 @@ class RawDisk(Evidence):
   """Evidence object for Disk based evidence.
 
   Attributes:
-    device_path (str): Path to a relevant 'raw' data source (ie: a block
+    source_path (str): Path to a relevant 'raw' data source (ie: a block
```

Comment:
small nit since this is just a docstring, but I think we still want `device_path` here since we set that in the pre-processors, right?  And `source_path` is already in the base object, so I don't think we need to document it again here.